src/gateway/bridge/mod.rs

```diff
@@ -61,16 +61,16 @@ pub use self::shard_runner_message::ShardRunnerMessage;
 #[cfg(feature = "voice")]
 pub use self::voice::VoiceGatewayManager;
 use super::ChunkGuildFilter;
+use crate::all::ShardInfo;
```

Comment:
```suggestion
use crate::modell::gateway::ShardInfo;
```
We should avoid importing from `crate::all` internally.

Reply:
Fixed